src/shared/types/state-and-params.ts

```diff
@@ -24,6 +24,10 @@ export type ItemStateAndParams = {
     state?: ItemState;
 };
 
+export type ItemStateAndParamsChangeOptions = {
+    action: 'setParams' | 'remove';
```

Comment:
May be better name 'removeParams' or 'removeActionParams' (depend on what we need to clear)

Reply:
Maybe removeItem?